tests/unit/jsonapi-vuex.spec.js

```diff
@@ -326,6 +258,21 @@ beforeEach(function() {
     }
   }
 
+  // Variables Used by required submodules.
+  // HACK: Rely on `this` so that submodules can access them. Eventually, we
+  // will want to define these in fixture files and import them as needed.
+  // Could use a factory library. See `fixtures/json_index_1.js`.
+  this.api = api;
```

Comment:
This will all go away if we continue with splitting up the tests into multiple files.